packages/lit-auth-client/src/lib/utils.ts

```diff
@@ -277,14 +286,14 @@ export function parseAuthenticatorData(
 // **`unparse()` - Convert UUID byte array (ala parse()) into a string**
 export function unparse(buf: any) {
```

Comment:
nit: do we have a unit test on this?

Reply:
We don't, tried adding it but it was a hell to set. Anyway those changes should not impact logic as are just naming or declaration keywords